### ISSUE TITLE
[BUG] VC Redis .dlls are not beeing copied 

### ISSUE THREAD

**Issue by rokk4** [YehudaKremer/msix#79]
### :information_source: Info

Version: `v2.8.4`

### :speech_balloon: Description

The VC .dlls are not getting copied.  I stumpled upon this by testing a released Store App on a freshly installed Win 10 VM ... fml...
Before the .dlls where always copied. I unpacked the .msix with the MSIX tool and could verify that no VC .dlls are included. :(

This seems to be a regression of #30 .  I can see that a test for this exists, so it seems very strange to me that this happens. I also verified that the VC libs are in the local msix plugin package.

BTW: Thank you very much for this package, it makes life much easier. 👍 

### :scroll: Pubspec.yaml
I use the --store flag to build for the store.

The configuration was working perfectly  up to 2.8.0 or maybe 2.8.2.

```yaml
msix_config:
  display_name: XXXXX®
  publisher_display_name: XXXX XXX XXXX
  identity_name:  XXXX.XXXXX
  msix_version: X.X.X.0
  publisher: CN=XXXXXXXXXXXXXXXXXXXXXXXXXX
  certificate_path: D:\XXXX\windows\msix\cert.pfx
  certificate_password: XXXXX
  logo_path: D:\XXXXXX\assets\icon\icon.png
  architecture: x64
  languages: "de-de"
  capabilities: 'internetClient'
```

**Comment by YehudaKremer** [comment]
Hello @rokk4 

I published fix version [2.8.5](https://pub.dev/packages/msix),
please check if this version solve this problem

Thank you for this critical report,
I'm happy to hear that this package is useful

**Comment by rokk4** [comment]
@YehudaKremer I just confirmed the .dlls get packed using the MSIX tool. :)

Thank very much for your fast fix! 🥇 


One friendly, opinionated thought I had looking through this package: 
I would try to avoid the use of async/streams  as much as possible in this batch program, since we have no FlutterUI worker getting janky, so there is no need for this sort of complexity IMHO.

Small example:
If there is a function which is awaiting all its async/calls or listening to a stream only once, it would consider it a code smell if not dealing with distributed systems. I would always try to move the async stuff inside the awaited function calls if possible.
```dart
  Future<void> copyVCLibsFiles() async {
    const taskName = 'copying VC libraries';
    _log.startingTask(taskName);

    _vCLibsFiles = _getAllDirectoryFiles(
        '${_config.vcLibsFolderPath()}/${_config.architecture}');

    for (File file in _vCLibsFiles) {
      await File(file.path)
          .copy('${_config.buildFilesFolder}/${basename(file.path)}');
    }
```
 using dart.io sync methods, I think most basic operations (read/write/delete) have sync versions. Maybe this could be utilized to get rid of some async stuff. :)
```dart
  void copyVCLibsFiles() {
    const taskName = 'copying VC libraries';
    _log.startingTask(taskName);

    _vCLibsFiles = _getAllDirectoryFiles(
        '${_config.vcLibsFolderPath()}/${_config.architecture}');

    for (File file in _vCLibsFiles) {
       File(file.path)
          .copySync('${_config.buildFilesFolder}/${basename(file.path)}');
    }
    _log.taskCompleted(taskName);
  }
```
But this is really just the opinion of someone who was traumtized by dealing with a lot of race conditions haha. 
Thanks again for this package!
